ConsentViewController/Classes/Constants.swift

```diff
@@ -16,8 +16,8 @@ let prod = (Bundle.framework.object(forInfoDictionaryKey: "SPEnv") as? String) !
 
 struct Constants {
     struct Urls {
-        static let envParam = prod ? "prod" : "stage"
-        static let SP_ROOT = URL(string: prod ? "https://cdn.privacy-mgmt.com/" : "https://cdn.sp-stage.net/")!
+        static let envParam = prod ? "localProd" : "stage"
+        static let SP_ROOT = URL(string: prod ? "http://localhost:3000/" : "https://cdn.sp-stage.net/")!
```

Comment:
Need to remember to change these back to prod before releasing 😅

Reply:
Right, maybe it would be better to replace them before merging into `V7`, I'll forget for sure!!